resources/find_latest_directory.py

```diff
@@ -0,0 +1,13 @@
+import glob
+import os
+import sys
+
+from os.path import exists, getmtime, join
+
+archive_dir = sys.argv[1]
+print(archive_dir)
+
+sub_dirs = glob.glob(join(archive_dir, '*'))
+if not sub_dirs:
+    print(None)
+print(max(sub_dirs, key=getmtime))
```

Comment:
I'd prefer if these scripts were importable from other Python files, to promote re-use.

E.g. the following (untested) code.

```python
import glob
import os
import sys

from os.path import exists, getmtime, join


def find_latest_directory(archive_dir):
    sub_dirs = glob.glob(join(archive_dir, "*"))
    if not sub_dirs:
        return None
    return max(sub_dirs, key=getmtime)


if __name__ == "__main__":
    archive_dir = sys.argv[1]
    print(find_latest_directory(archive_dir))
```

Reply:
But... should we be using the directory with the latest build number, rather than last access time? I think we did something similar for the feed build logic.

Reply:
I can get onboard with the import of things. I didn't argparse either, but since it's taking simple arguments, I didn't really care too much.

For the feed build, we did that because of different versions. Say we were trying to pull in 20.3, but something in 20.2 got rebuilt, we would accidentally pull in 20.2. For this, we wouldn't have that explicit problem because we don't build older builds of the same version anymore. I suppose someone could manually mess with the exported files and we could hit the same thing, but I'm not as concerned about that. If you are, I can change it.

Reply:
> I didn't argparse either, but since it's taking simple arguments, I didn't really care too much.

Completely agree.

> I suppose someone could manually mess with the exported files and we could hit the same thing, but I'm not as concerned about that. If you are, I can change it.

I'm fine with the existing solution for now.

> For this, we wouldn't have that explicit problem because we don't build older builds of the same version anymore.

Thanks for reminding me what the rationale behind the other approach was!